src/codeu/chat/server/Model.java

```diff
@@ -134,6 +142,24 @@ public void add(Message message) {
   public StoreAccessor<String, Message> messageByText() {
     return messageByText;
   }
+<<<<<<< HEAD
+
+  public void changeAuthority(ConversationHeader conversation, Uuid targetUser, byte authorityByte)
```

Comment:
Don't forget - you should validate that the user has the authority to call this method! I don't believe you are currently checking it.